Source/ACE.Server/Program.cs

```diff
@@ -149,6 +147,8 @@ public static void Main(string[] args)
             log.Info("Initializing ConfigManager...");
             ConfigManager.Initialize();
 
+            Console.Title = @$"{ConfigManager.Config.Server.WorldName} - v{ServerBuildInfo.FullVersion}";
```

Comment:
```suggestion
            Console.Title = @$"ACEmulator - v{ServerBuildInfo.FullVersion}:  {ConfigManager.Config.Server.WorldName}";
```

Reply:
this will show ACEmulator - v#: ACEmulator for the default scenario, which seems kind of messy

We should decide on some kind of definitive formatting though, since Window title is something important